pattern-distributed/confs/sp-worker/conf/deployment.yaml

```diff
@@ -1,5 +1,5 @@
 ################################################################################
-#   Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```

Comment:
Let's not update the licence years

Reply:
This is the license header in the pack. If it is incorrect we need to change that as well, isn't it?